tgui/packages/tgui/interfaces/LongRangeTeleporter.js

```diff
@@ -0,0 +1,49 @@
+import { useBackend } from '../backend';
+import { Button, LabeledList, Section } from '../components';
+import { Window } from '../layouts';
+
+export const LongRangeTeleporter = (props, context) => {
+  const { act, data } = useBackend(context);
+
+  const {
+    destinations,
+    receive_allowed,
+    send_allowed,
+    syndicate,
+  } = data;
+
+  return (
+    <Window
+      theme={syndicate ? 'syndicate' : 'ntos'}
+      width={390}
+      height={380}>
+      <Window.Content>
+        <Section title="Destinations">
+          <LabeledList>
+            {destinations.length ? destinations.map((d) => (
+              <LabeledList.Item label={d["destination"]} key={d["ref"]}>
```

Comment:
"ref" is null for a subset of destinations, which would mean the same key for them. Consider `${ref ?? name}` or some compound key (i.e. use both).

Reply:
Compound it is!